sql/1.027/updates_table.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+CREATE TABLE "instant_answer_updates" (
+      "id" text NOT NULL,
+      "field" text NOT NULL,
+      "value" text NOT NULL,
+      "timestamp" text NOT NULL,
+      PRIMARY KEY ("id")
+);
+
+ALTER TABLE "instant_answer_updates" ADD CONSTRAINT "instant_answer_updates_fk_id" FOREIGN KEY ("id")
+  REFERENCES "instant_answer" ("id") ON DELETE CASCADE ON UPDATE CASCADE DEFERRABLE;
```

Comment:
Yeah, looks like this DDL for this is wrong:

https://github.com/duckduckgo/community-platform/blob/jd/updates_table/lib/DDGC/DB/Result/InstantAnswer/Updates.pm#L32

It should probably have a primary key also, even if that's also `instant_answer_id`

Running `ddgc_schema_diff` gives me:

```sql
CREATE TABLE "instant_answer_updates" (
  "instant_answer_id" text NOT NULL,
  "field" text NOT NULL,
  "value" text NOT NULL,
  "timestamp" text NOT NULL
);

ALTER TABLE "instant_answer_updates" ADD CONSTRAINT "instant_answer_updates_fk_instant_answer_id" FOREIGN KEY ("instant_answer_id")
  REFERENCES "instant_answer" ("id") ON DELETE CASCADE ON UPDATE CASCADE DEFERRABLE;
```